src/CfgEditor/ICfgEditorPanel.ts

```diff
@@ -0,0 +1,25 @@
+import * as vscode from "vscode";
+
+export interface ICfgEditorPanel extends vscode.CustomTextEditorProvider {
+
+    resolveCustomTextEditor(
```

Comment:
It seems ICfgEditorPanel has no additional definitions or interface, so it's working just as an alias of vscode.CustomTextEditorProvider. Do you plan to add something here?

Reply:
> So, in fact, during writting this comment, I think ICfgEditorPanel is useless file😓

I am not sure about your plan, so I wanna ask if you plan to modify this file or keep it..? :thinking:

Reply:
Actually Nope.
There is no plan😕

Reply:
If so, how about removing this one and using vscode.CustomTextEditorProvider, just like before?

Reply:
I also think that's the better idea😊
Then, I will remove this code and make a new commit which includes `EdgeTPUCfgEditorPanel.ts` that implements vscode.CustomTextEditorProvider!

In this case, I plan to commit in this PR, but should I close this pull request and open a new one?

Reply:
> In this case, I plan to commit in this PR, but should I close this pull request and open a new one?

You can choose whatever for your own convenience  :-D

Reply:
Then I will make a new commit here, thanks!